lib/zold/node/farm.rb

```diff
@@ -165,11 +165,20 @@ def save(threads, list = [])
     def load
       @mutex.synchronize do
         if File.exist?(@cache)
-          AtomicFile.new(@cache).read.split(/\n/).map { |t| Score.parse(t) }
+          AtomicFile.new(@cache).read.split(/\n/)
+            .map { |t| parse_score_line(t) }
+            .reject(&:zero?)
         else
           []
         end
       end
     end
+
+    def parse_score_line(line)
+      Score.parse(line)
+    rescue StandardError => e
+      @log.error(e.message)
```

Comment:
@emilianodellacasa see how `Backtrace` is used for this purposed in other places of the app